27_Civil_War/civilwar.bas

```diff
@@ -216,7 +216,7 @@
 2280  PRINT "CASUALTIES",C5,C6
 2290  PRINT "DESERTIONS",INT(E),INT(E2)
 2300  PRINT
-2310  IF B$ <> "YES" THEN 2350
+2310  IF B$ <> "YES" THEN 2530
```

Comment:
2530 is correct according to the listing at https://www.atariarchives.org/basicgames/showpage.php?page=49